include/geolib/sensors/LaserRangeFinder.h

```diff
@@ -129,6 +130,21 @@ class LaserRangeFinder {
 
     // Number of beams in a half circle
     uint i_half_circle_;
+    
+    /**
+     * @brief Get the index of the first beam with a higher angle than this beam
+     * @param angle radial angle of the beam
+     * @return Index of the beam
+     */
+    int getAngleUpperIndexRaw(double angle) const;
```

Comment:
I think these should be public.

Reply:
Right now, these aren't used outside this class. So I don't see a reason why. We can change it when it is needed.